bot/code_review_bot/report/base.py

```diff
@@ -44,6 +44,9 @@
 The analysis task [{name}]({url}) failed, but we could not detect any issue.
 Please check this task manually.
 """
+FRONTEND_LINKS = """
+You can view these defects on [the code-review frontend]({frontend_url}) and on [Treeherder]({treeherder_url}).
```

Comment:
Nit: any idea for a better name than `code-review frontend`?

Reply:
Nope. Just use the domain name `code-review.moz.tools` ?

Reply:
Let's keep `code-review frontend` then